WinrtServer/CreateCppInterop.bat

```diff
@@ -1,7 +1,12 @@
 set OutDir=%1
-if not exist %OutDir%Include\WinrtServer (
-    mkdir %OutDir%Include\WinrtServer || exit /b 1
+
+if exist %OutDir%Include\WinrtServer\winrt\ (
+    del /s /q %OutDir%Include\WinrtServer\winrt\ || exit /b 1
 )
-winmdidl %OutDir%\WinrtServer.winmd /outdir:%OutDir%Include\WinrtServer\ || exit \b 1
-pushd %OutDir%Include\WinrtServer || exit /b 1
-midlrt WinrtServer.idl /metadata_dir C:\Windows\System32\WinMetadata /ns_prefix || exit /b 1
\ No newline at end of file
+
+REM Copy the primary WinrtServer.h. This is a workaround for include path issues in generated files
+xcopy /I "Generated Files\winrt\WinrtServer.h" %OutDir%Include\WinrtServer\
```

Comment:
I think you can get rid of this extra xcopy if you change the client code includes to `#include <winrt/WinrtServer.h> (or similar)`